server/apps/immich/src/api-v1/asset/asset.controller.ts

```diff
@@ -50,13 +50,13 @@ import {
   IMMICH_CONTENT_LENGTH_HINT,
 } from '../../constants/download.constant';
 
-@Authenticated()
```

Comment:
same question as before, is it possible to only override the few methods with `{ isShared: true }` annotations?